pkg/yurthub/filter/servicetopology/filter.go

```diff
@@ -81,6 +82,7 @@ func (stf *serviceTopologyFilter) SupportedResourceAndVerbs() map[string]sets.Se
 }
 
 func (stf *serviceTopologyFilter) SetSharedInformerFactory(factory informers.SharedInformerFactory) error {
+	factory.Core().V1().Services().Informer().SetTransform(util.TransformStripManagedFields())
```

Comment:
how about move `SetTransform` into `Initialize` of SharedInformer in `cmd/yurthub/app/config/config.go` file? you can create a new `informer` for `Service` resource.

Reply:
fixed